src/sessions/PageManager.ts

```diff
@@ -8,6 +8,7 @@ export type Page = {
     pageId: string;
     interaction: number;
     parentPageId?: string;
+    timeSpentOnParentPage?: number;
```

Comment:
question: Is there a way we can omit `timeOnParentPage` from this structure? This is stored on browser storage, so if we don't need it, we should omit it.

I feel like it's something we can compute during `recordPageView()`.

Reply:
Fixed in revision. Now its done during ```recordPageView```